openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -545,14 +545,23 @@ def _handle_duplicate_email_username(self, request, data):
         username = data.get('username')
         errors = {}
 
+        # TODO: remove the is_authn_mfe check and use the new error message as default after redesign
+        is_authn_mfe = data.get('is_authn_mfe', False)
```

Comment:
We can avoid this if we just start sending the error codes to front-end alongside old messages and manage those errors at the front-end.

Reply:
error codes are used currently to set the error messages for alert box not for field messages but yes I can look into it